src/main/kotlin/step3/domain/race/RaceResult.kt

```diff
@@ -0,0 +1,11 @@
+package step3.domain.race
+
+data class RaceResult(val roundResultList: List<RaceRoundResult>) {
+
+    fun getRoundCount(): Int = roundResultList.size
```

Comment:
객체의 상태를 나타낸다면 프로퍼티로 표현해주셔도 좋을 것 같아요. 
```suggestion
val roundCount: Int = roundResult.size
```